### ISSUE TITLE
Google Maps on recommended list

### ISSUE THREAD

**Issue by btTeddy** [0x192/Universal-Android-Debloater#107]
Samsung Galaxy S21

Google maps removed by default

gMmaps is a major app used by almost the whole world, shouldn't be really on default list (same as youtube TBH)

**Comment by devanks** [comment]
I think the question is whether the app is absolutely required for your phone to function and not whether you would use it or not.
I normally prefer to have these apps uninstalled by the script and then install from Play Store if necessary.
So until the disabling/uninstalling of the apps creates an instability, this cannot be an issue IMO.

**Comment by btTeddy** [comment]
Fair enough but uninstalling it from "recommended" list will be seing by
many as breaking core functionality.

The most default "recommended" list should remove only apps not causing any
issues, neither removing any functionality - basically a bloatware.

**Comment by devanks** [comment]
I think it's a personal preference. In my mind most or all apps which are not essential for me to use my phone installed from the beginning is bloatware.
And Google Maps and YouTube fits the description. You could easily prefer to install a modded version of YouTube or maybe prefer to not have Google take your data from the Maps app. So they are bloatware IMO. I get your point though, because I do keep Google maps around.
Maybe the dev or someone could do a survey to confirm what the users needs because at this point this is just both of our opinions.

**Comment by KarlRamstedt** [comment]
I'm with @devanks on this one.
They're not important for basic functionality ***and can easily be reinstalled through Play Store***.
What makes most Google apps extra juicy to disable/uninstall is the fact that most of them occasionally run in the background, staying cached in RAM most of the time, for no good reason.

**Comment by genodeftest** [comment]
Same here. I think it is bloatware because I prefer [OsmAnd+](https://play.google.com/store/apps/details?id=net.osmand.plus).

**Comment by ruralTom** [comment]
You can install it as a user app. Google's apps are almost the entire reason to debloat a phone.
